service/apigateway/serializers.go

```diff
@@ -4371,20 +4369,6 @@ func awsRestjson1_serializeOpHttpBindingsGetExportInput(v *GetExportInput, encod
 	return nil
 }
 
-func awsRestjson1_serializeOpDocumentGetExportInput(v *GetExportInput, value smithyjson.Value) error {
-	object := value.Object()
-	defer object.Close()
-
-	if v.Parameters != nil {
-		ok := object.Key("parameters")
-		if err := awsRestjson1_serializeDocumentMapOfStringToString(v.Parameters, ok); err != nil {
-			return err
-		}
-	}
```

Comment:
Was this incorrectly being serialized to the payload body before this change?  Looks like so, because this is a GET operation.

Reply:
Yeah looks like it, this change should fix it.